Orchestrator/Prognetics.Saga.Orchestrator/SagaLog/MongoDbCompensationStore.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MongoDB.Driver;
+using Prognetics.Saga.Core.Abstract;
+using Prognetics.Saga.Core.Model;
+
+namespace Prognetics.Saga.Orchestrator.SagaLog;
+
+public class MongoDbCompensationStore : ICompensationStore
```

Comment:
As above. Specific implementation of the SagaLog for Mongo should be a separate DLL. Just like parsers are.